cli/src/helper.rs

```diff
@@ -195,7 +195,12 @@ pub async fn import_mnemonic(path: &str) -> Result<String, Error> {
 }
 
 async fn write_mnemonic_to_file(path: &str, mnemonic: &str) -> Result<(), Error> {
-    let mut file = OpenOptions::new().create(true).append(true).open(path).await?;
+    let mut file = OpenOptions::new()
+        .create(true)
+        .append(true)
+        .mode(0o600)
```

Comment:
what happens on Windows?

Reply:
Oh actually this is the tokio version, it doesn't even exist on Windows my bad, just put a cfg

Reply:
okay, then we need to have specific code depending on the platform we're running on.

Reply:
Good point, I was hoping the `tokio` has some mappings from posix. Apprently windows have a different set of methods.

Reply:
Yeah, something like they do in this example https://doc.rust-lang.org/std/os/unix/fs/trait.OpenOptionsExt.html#examples-1

Reply:
fixed

Reply:
So what about windows?

Reply:
This is a good question. I've been trying to figure out what would be best for Windows. But honestly, I haven't used this system for many years and their permission model is unknown to me, and don't have the chance to test it. I would suggest creating another story for Windows as it requires additional research. But if it's necessary I could try to look at this closer.

Reply:
Can you open an issue please?